scripting/executes.sp

```diff
@@ -654,7 +654,7 @@ public void EndWarmup() {
                         g_hMinPlayers.IntValue);
   ServerCommand("mp_death_drop_gun 1");
   ServerCommand("mp_warmup_pausetimer 0");
-  ServerCommand("mp_warmuptime 10");
+  ServerCommand("mp_warmuptime 3");
```

Comment:
Is this change necessary to fix the issue, or is this a style/gameplay choice? IMO, 3 seconds is a bit short.

Reply:
I think the main reason around this bug is a race condtion. we are just escaping that race condition tbh. 
and i always wondered why you force some commands. specially warmuptime . isn't it better to make them added in a cfg rather than forced by a plugin?

Reply:
> isn't it better to make them added in a cfg rather than forced by a plugin?

Yes. It's just more overhead and configuration that can become wrong (e.g. file is missing, user adds commands that triggers some bug), so I didn't bother here.

Reply:
well in that case. i guess this can be changed to 5 seconds. and then its all good to merge ?. i prefer 3 seconds personally.